stdlib/src/main/scala/testz/stdlib.scala

```diff
@@ -74,6 +74,51 @@ object PureSuite {
   }
 }
 
+abstract class ImpureSuite extends Suite {
+  def test[T](test: Test[λ[A => () => Future[A]], () => Future[T]]): T
```

Comment:
what's the point of having the `Future` lifted into a `() => ?`? It's not like it's pure either way.

Reply:
a) This is asynchronous and only requires the stdlib.
b) This is as pure as `Task` ;)